<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createClass</class>
 <widget class="QWidget" name="createClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Class</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Create New Class</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <widget class="QPushButton" name="manualEntryNamePushButton">
        <property name="text">
         <string>Manual Entry</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="savePushButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="importPushButton">
        <property name="text">
         <string>Import Roster</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLineEdit" name="studentNameLineEdit"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="classNameLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="classNameLabel">
        <property name="text">
         <string>Class Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableView" name="studentTableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
